test/s3-benchmarks/src/main/java/software/amazon/awssdk/s3benchmarks/BaseTransferManagerBenchmark.java

```diff
@@ -95,36 +93,15 @@ protected void additionalWarmup() {
 
     protected abstract void doRunBenchmark();
 
-    protected final void printOutResult(List<Double> metrics, String name) {
-        logger.info(() -> String.format("===============  %s Result ================", name));
-        logger.info(() -> "" + metrics);
-        double averageLatency = metrics.stream()
-                                       .mapToDouble(a -> a)
-                                       .average()
-                                       .orElse(0.0);
-
-        double lowestLatency = metrics.stream()
-                                      .mapToDouble(a -> a)
-                                      .min().orElse(0.0);
-
-        HeadObjectResponse headObjectResponse = s3Sync.headObject(b -> b.bucket(bucket).key(key));
-        double contentLengthInGigabit = (headObjectResponse.contentLength() / (1000 * 1000 * 1000.0)) * 8.0;
-        logger.info(() -> "Average latency (s): " + averageLatency);
-        logger.info(() -> "Object size (Gigabit): " + contentLengthInGigabit);
-        logger.info(() -> "Average throughput (Gbps): " + contentLengthInGigabit / averageLatency);
-        logger.info(() -> "Highest average throughput (Gbps): " + contentLengthInGigabit / lowestLatency);
-        logger.info(() -> "==========================================================");
-    }
-
     private void cleanup() {
         s3Sync.deleteObject(b -> b.bucket(bucket).key(WARMUP_KEY));
-        transferManager.close();
+        s3.close();
     }
 
     private void warmUp() throws InterruptedException {
         logger.info(() -> "Starting to warm up");
 
-        for (int i = 0; i < WARMUP_ITERATIONS; i++) {
+        for (int i = 0; i < PRE_WARMUP_ITERATIONS; i++) {
```

Comment:
"warmup" or "pre-warmup"? The latter seems to imply:
1. pre-warmup
1. warmup
1. actual run

Reply:
I think for the sake of simplicity, I would still consider merging all of this into the pure "warmup" phase, and running more iterations if needed.